02/app.js

```diff
@@ -4,12 +4,22 @@ import { createRoot } from 'react-dom/client';
 const root = createRoot(document.querySelector('#root'));
 
 class Counter extends React.Component {
-    state = {
-        amount: 0,
+    constructor(props) {
+        super(props)
+        this.state = {
+            amount: 0,
+        }
+        this.handleCLick = this.handleCLick.bind(this)
```

Comment:
Jeśli używać inicjalizatora właściwości (linia 15) to ten zapis nie jest potrzebny.